add a project ATM in project directory (#21)
computerclubkec/oop: projects/README.md

## Existing Projects:

1. **[WhatsThatFigure](https://github.com/gaurovgiri/whatsthatfigure)**: A multiplayer pictionary game using QT framework.

> 2. **[ATM](https://github.com/aayush105/ATM)**: This is a simple C++ ATM console application for basic banking operations with file-based data storage.

Reviewer: ```suggestion
2. **[ATM](https://github.com/aayush105/ATM)**: A simple C++ ATM console application for basic banking operations with file-handling.
```
